src/server/lesson-controller.js

```diff
@@ -26,7 +26,7 @@ const { NotFound, BadRequest, Forbidden, Unauthorized } = httpErrors;
 
 const isRoomOwnerOrCollaborator = ({ room, userId }) => {
   const isOwner = room.owner === userId;
-  const isCollaborator = room.lessonsMode === ROOM_LESSONS_MODE.collaborative && room.members.find(m => m.userId === userId);
+  const isCollaborator = room.lessonsMode === ROOM_LESSONS_MODE.collaborative && room.members.some(m => m.userId === userId);
```

Comment:
The last place to use `find` would be line 67 in this file ;-)

Reply:
As in 'it should be `some` there too'? If so, updated, if not, I need more info 🙏

Reply:
Yes exactly (although not really very important), thanks!

Reply:
I think it's important cause we could end up with surprises if we rely on a boolean value. Thanks for pointing it out.